modules/support/src/evm.rs

```diff
@@ -121,6 +121,19 @@ impl<AccountId, Balance: Default> EVMBridge<AccountId, Balance> for () {
 	fn set_origin(_origin: AccountId) {}
 }
 
+/// An abstraction of EVMChainId
+pub trait EVMChainId {
```

Comment:
do we need a new trait or can we just use `Get<u64>`? we can have a new `struct EvmChainId` implements it.